x-pack/dockerlogbeat/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM  golang:latest as builder
```

Comment:
See `.go-version` file for current go version in the repository.

Reply:
Is there some way to access that from mage?

Reply:
`dev-tools/mage/settings.go` exports GoVersion

Reply:
Thanks!